website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+// All of the following configs are used in the website and documentation provided by the Design team Glyph
+
+// Colors
+
+@mixin detox-colors($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
```

Comment:
On the other hand, I see that you wanted to avoid repetitiveness. Okay, fine, I understand, but there are other options if you really want that.

See a nice example here: https://stackoverflow.com/a/61191193